### PR TITLE
chore: indicate staleness more prominently in `next info` output

### PR DESCRIPTION
**Pull request by balazsorban44**
### What?

Improve the `next info` output.

<details>
<summary>Before:
</summary>

</details>

<details>
<summary>After:
</summary>

</details>

### Why?

People have been ignoring the ask to check `next@canary` before opening a new issue, which results in triaging overhead. This PR inlines the warning into `next info` that would make it an extra effort to remove the warning without reading the message, hopefully leading to not ignoring the message anymore.

### How?

Reusing the same logic from the version staleness indicator from Error Overlay.

[Slack thread](https://vercel.slack.com/archives/C03KAR5DCKC/p1704364764719949)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1.bug_report.yml [hunk 1]
```diff
@@ -40,13 +40,6 @@ body:
       placeholder: 'Following the steps from the previous section, I expected A to happen, but I observed B instead'
     validations:
       required: true
-  - type: checkboxes
```
Comment (balazsorban44) [inline .github/ISSUE_TEMPLATE/1.bug_report.yml:43]: Let me know if we should still leave this or not.

**Comment by ijjk** [comment]
# Stats from current PR

<details open>
<summary><strong>Default Build</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | vercel/next.js chore/tweak-cli-info-output | Change |
| - | - | - | - |
| buildDuration | 13.4s | 13.2s | N/A |
| buildDurationCached | 7.5s | 6.6s | N/A |
| nodeModulesSize | 201 MB | 200 MB | N/A |
| nextStartRea..uration (ms) | 413ms | 413ms | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js chore/tweak-cli-info-output | Change |
| - | - | - | - |
| 193.HASH.js gzip | 181 B | 182 B | N/A |
| 3f784ff6-HASH.js gzip | 53.3 kB | 53.3 kB | N/A |
| 433-HASH.js gzip | 28.6 kB | 28.6 kB | N/A |
| framework-HASH.js gzip | 45.2 kB | 45.2 kB | ✓ |
| main-app-HASH.js gzip | 241 B | 242 B | N/A |
| main-HASH.js gzip | 31.7 kB | 31.8 kB | N/A |
| webpack-HASH.js gzip | 1.7 kB | 1.7 kB | N/A |
| Overall change | 45.2 kB | 45.2 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js chore/tweak-cli-info-output | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js chore/tweak-cli-info-output | Change |
| - | - | - | - |
| _app-HASH.js gzip | 194 B | 195 B | N/A |
| _error-HASH.js gzip | 183 B | 181 B | N/A |
| amp-HASH.js gzip | 504 B | 502 B | N/A |
| css-HASH.js gzip | 321 B | 321 B | ✓ |
| dynamic-HASH.js gzip | 2.5 kB | 2.5 kB | N/A |
| edge-ssr-HASH.js gzip | 255 B | 253 B | N/A |
| head-HASH.js gzip | 350 B | 349 B | N/A |
| hooks-HASH.js gzip | 369 B | 369 B | ✓ |
| image-HASH.js gzip | 4.28 kB | 4.28 kB | N/A |
| index-HASH.js gzip | 255 B | 256 B | N/A |
| link-HASH.js gzip | 2.61 kB | 2.61 kB | ✓ |
| routerDirect..HASH.js gzip | 312 B | 311 B | N/A |
| script-HASH.js gzip | 385 B | 383 B | N/A |
| withRouter-HASH.js gzip | 307 B | 308 B | N/A |
| 1afbb74e6ecf..834.css gzip | 106 B | 106 B | ✓ |
| Overall change | 3.4 kB | 3.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js chore/tweak-cli-info-output | Change |
| - | - | - | - |
| _buildManifest.js gzip | 483 B | 484 B | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | vercel/next.js chore/tweak-cli-info-output | Change |
| - | - | - | - |
| index.html gzip | 528 B | 525 B | N/A |
| link.html gzip | 541 B | 540 B | N/A |
| withRouter.html gzip | 525 B | 523 B | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Edge SSR bundle Size</strong></summary>

|  | vercel/next.js canary  | vercel/next.js chore/tweak-cli-info-output | Change |
| - | - | - | - |
| edge-ssr.js gzip | 93.8 kB | 93.8 kB | N/A |
| page.js gzip | 148 kB | 147 kB | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Middleware size</strong></summary>

|  | vercel/next.js canary  | vercel/next.js chore/tweak-cli-info-output | Change |
| - | - | - | - |
| middleware-b..fest.js gzip | 623 B | 623 B | ✓ |
| middleware-r..fest.js gzip | 151 B | 151 B | ✓ |
| middleware.js gzip | 37.4 kB | 37.4 kB | N/A |
| edge-runtime..pack.js gzip | 1.92 kB | 1.92 kB | ✓ |
| Overall change | 2.7 kB | 2.7 kB | ✓ |

</details>

<details>
<summary><strong>Next Runtimes</strong></summary>

|  | vercel/next.js canary  | vercel/next.js chore/tweak-cli-info-output | Change |
| - | - | - | - |
| app-page-exp...dev.js gzip | 169 kB | 169 kB | N/A |
| app-page-exp..prod.js gzip | 95.2 kB | 94.2 kB | N/A |
| app-page-tur..prod.js gzip | 95.8 kB | 94.9 kB | N/A |
| app-page-tur..prod.js gzip | 90.4 kB | 89.5 kB | N/A |
| app-page.run...dev.js gzip | 142 kB | 139 kB | N/A |
| app-page.run..prod.js gzip | 89.8 kB | 88.8 kB | N/A |
| app-route-ex...dev.js gzip | 24.1 kB | 24.1 kB | N/A |
| app-route-ex..prod.js gzip | 16.7 kB | 16.7 kB | ✓ |
| app-route-tu..prod.js gzip | 16.7 kB | 16.7 kB | ✓ |
| app-route-tu..prod.js gzip | 16.3 kB | 16.3 kB | ✓ |
| app-route.ru...dev.js gzip | 23.5 kB | 23.5 kB | N/A |
| app-route.ru..prod.js gzip | 16.3 kB | 16.3 kB | ✓ |
| pages-api-tu..prod.js gzip | 9.38 kB | 9.38 kB | ✓ |
| pages-api.ru...dev.js gzip | 9.65 kB | 9.65 kB | ✓ |
| pages-api.ru..prod.js gzip | 9.37 kB | 9.37 kB | ✓ |
| pages-turbo...prod.js gzip | 21.9 kB | 21.9 kB | ✓ |
| pages.runtim...dev.js gzip | 22.6 kB | 22.6 kB | ✓ |
| pages.runtim..prod.js gzip | 21.9 kB | 21.9 kB | ✓ |
| server.runti..prod.js gzip | 49.5 kB | 49.5 kB | ✓ |
| Overall change | 210 kB | 210 kB | ✓ |

</details>

<details>
<summary><strong>Diff details</strong></summary>

<details>
<summary>Diff for <strong>page.js</strong></summary>

Diff too large to display
</details>
<details>
<summary>Diff for <strong>edge-ssr.js</strong></summary>

Diff too large to display
</details>
<details>
<summary>Diff for <strong>3f784ff6-HASH.js</strong></summary>

Diff too large to display
</details>
<details>
<summary>Diff for <strong>app-page-exp..ntime.dev.js</strong></summary>

```diff
failed to diff
```
</details>
<details>
<summary>Diff for <strong>app-page-exp..time.prod.js</strong></summary>

Diff too large to display
</details>
<details>
<summary>Diff for <strong>app-page-tur..time.prod.js</strong></summary>

Diff too large to display
</details>
<details>
<summary>Diff for <strong>app-page-tur..time.prod.js</strong></summary>

Diff too large to display
</details>
<details>
<summary>Diff for <strong>app-page.runtime.dev.js</strong></summary>

```diff
failed to diff
```
</details>
<details>
<summary>Diff for <strong>app-page.runtime.prod.js</strong></summary>

Diff too large to display
</details>
<details>
<summary>Diff for <strong>server.runtime.prod.js</strong></summary>

Diff too large to display
</details>

</details>

</details>
Commit: bd018e66d09f3e9ec06e7479f2d7f9e100531c3d

**Comment by ijjk** [comment]
## Failing test suites 
Commit: bd018e66d09f3e9ec06e7479f2d7f9e100531c3d

`pnpm test-dev test/e2e/app-dir/navigation/navigation.test.ts` 
- app dir - navigation > query string > useParams identity between renders > should be stable in pages

<details>
<summary>Expand output</summary>

  ● app dir - navigation › query string › useParams identity between renders › should be stable in pages

    TIMED OUT: success

    fail

    undefined

      626 |
      627 |   if (hardError) {
    > 628 |     throw new Error('TIMED OUT: ' + regex + '\n\n' + content + '\n\n' + lastErr)
          |           ^
      629 |   }
      630 |   return false
      631 | }

      at check (lib/next-test-utils.ts:628:11)
      at runTests (e2e/app-dir/navigation/navigation.test.ts:88:11)
      at Object.<anonymous> (e2e/app-dir/navigation/navigation.test.ts:113:11)

</details>


Read more about building and testing Next.js in [contributing.md](https://github.com/vercel/next.js/blob/canary/contributing.md).

File: packages/next/src/cli/next-info.ts [hunk 2]
```diff
@@ -104,6 +108,35 @@ async function printDefaultInfo() {
   const installedRelease = getPackageVersion('next')
   const nextConfig = await getNextConfig()
 
+  let staleness = ''
+  let title = ''
+  let versionInfo
```
Comment (styfle) [inline packages/next/src/cli/next-info.ts:113]: Is `staleness` different than `versionInfo.staleness`? Looks like it so perhaps it should have a different name.

**Review by samcx — APPROVED** [review]
<picture data-single-emoji=":lgtm:" title=":lgtm:">:lgtm:</picture>

**Pull request merged**